homeassistant/components/airzone/climate.py

```diff
@@ -129,7 +130,11 @@ def __init__(
         super().__init__(coordinator, entry, system_zone_id, zone_data)
 
         self._attr_unique_id = f"{self._attr_unique_id}_{system_zone_id}"
-        self._attr_supported_features = ClimateEntityFeature.TARGET_TEMPERATURE
+        self._attr_supported_features = (
+            ClimateEntityFeature.TARGET_TEMPERATURE
+            | ClimateEntityFeature.TURN_OFF
+            | ClimateEntityFeature.TURN_ON
```

Comment:
The actual modes are determined later on... also, there are no turn_on/turn_off methods implemented.

How could you know on/off is supported at this point?

Reply:
> The actual modes are determined later on... also, there are no turn_on/turn_off methods implemented.
> 
> How could you know on/off is supported at this point?

All Airzone devices support turning them on/off regardless of their HVAC modes, and there are async_turn_on and async_turn_off methos defined:
https://github.com/home-assistant/core/blob/0afd8cfb5735784b26ae78e00de4faf8b8af28c3/homeassistant/components/airzone/climate.py#L178C15-L190

This change is correct.

Reply:
Thanks for clarifying @Noltari 👍 

../Frenck